cli/podman.go

```diff
@@ -77,13 +77,21 @@ type PodmanRemoveReport struct {
 	Stderr string
 }
 
+type PodmanUnshareReport struct {
+	Stdout string
+	Stderr string
+}
+
 type PodmanEngine interface {
+	Create(rawImage string, opts *PodmanCreateOptions) (*PodmanCreateReport, error)
+	CopyFrom(containerID, sourcePath, destinationPath string) (*PodmanCopyReport, error)
 	InspectImage(rawImage string, opts ImageInspectOptions) (*ImageInspectReport, error)
+	Mount(containerId string) (string, error)
 	Pull(rawImage string, opts ImagePullOptions) (*ImagePullReport, error)
+	Remove(containerID string) (*PodmanRemoveReport, error)
 	Run(opts ImageRunOptions) (*ImageRunReport, error)
 	Save(nameOrID string, tags []string, opts ImageSaveOptions) error
 	ScanImage(image string) (*ImageScanReport, error)
-	Create(rawImage string, opts *PodmanCreateOptions) (*PodmanCreateReport, error)
-	CopyFrom(containerID, sourcePath, destinationPath string) (*PodmanCopyReport, error)
-	Remove(containerID string) (*PodmanRemoveReport, error)
```

Comment:
Should we leave these as implemented functionality, even if we prefer to use unmount and unshare?

Reply:
I just reordered them. Alphabetically. :)

Reply:
Oh, whoops! I can't read. Resolved!